X9.146/gen_dilithium3_dual_keysig_cert.c

```diff
@@ -0,0 +1,379 @@
+/* custom_ext.c
```

Comment:
I can't see significant difference from level 2 version.
Can we make it that the level is a command line argument?

Reply:
Yes, I think we can do that.  will be in next update.